### ISSUE TITLE
Use python iRODS client to retrieve metadata and file info

### ISSUE THREAD

**Issue by hyi** [hydroshare/hydroshare#3744]
Currently we use icommands to not only do file i/o and transfer operations but also retrieve metadata and file info by parsing command output to retrieve needed info which depends on icommands output syntax. To make code more robust, python irods client should be used for non file transfer and i/o operations for now until python irods support multipart data object transfer which is on irods roadmap: https://irods.org/roadmap/

**Comment by Castronova** [comment]
@hyi Is this ticket still relevant or should it be closed?

**Comment by hyi** [comment]
@Castronova Yes, I updated the title to capture what needs to be done more accurately.
